lib/site_prism/element_checker.rb

```diff
@@ -3,15 +3,29 @@
 module SitePrism
   module ElementChecker
     def all_there?
-      elements_to_check.all? { |element| send "has_#{element}?" }
+      elements_to_check.all? { |element| present?(element) }
```

Comment:
some syntax sugar?
```ruby
elements_to_check.all?(&method(:present?))
```

Reply:
Genuine question. Which is more common / preferred? I never knew you could do this, I thought you had to write a proc.

You've probably got more experience than me, so not fussed either way. I personally haven't seen this used very often. So don't know.